### PR TITLE
Add DB class

### PR DESCRIPTION
**Pull request by DomBelcher**
This adds a database class wrapper for AWS DynamoDB, to be used to update the Alma cache database.

It adds methods to get, save and delete records. The get method includes a check that retrieved records have not expired.

### CODE REVIEW

File: src/db.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+require('aws-sdk/clients/dynamodb')
+const AWS = require('aws-sdk/global')
+
+class DB {
+  constructor (tableName, region = 'eu-west-2') {
```
Comment (stephenrob) [inline src/db.js]: Is this just setting a default value for the region to eu-west-2?

File: src/db.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    return this.docClient.put(params).promise()
+  }
+
+  get (Key) {
+    const params = {
+      TableName: this.tableName,
+      Key
+    }
+
+    const timeNow = Math.floor(Date.now() / 1000)
```
Comment (stephenrob) [inline src/db.js]: Could this be extracted to a helper method? It seems like a "Magic String"

**Review by stephenrob — CHANGES_REQUESTED** [review]
Couple of minor changes, around method arguments and a 'magic method/string'.

File: src/db.js (hunk 1 above)
Comment (DomBelcher) [reply src/db.js]: I forgot I left that in, it's just a default value, passing in a second argument to the method overrides it. Not sure if it's best left in or removed

Comment (stephenrob) [reply src/db.js]: For now i'd remove it as a default value as its not the common default value of AWS.

**Review by stephenrob — APPROVED** [review]

**Pull request merged**